### PR TITLE
Fixed typos in Quick Start guide; added information about JMS serializer

### PR DESCRIPTION
**Pull request by coyl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,17 +7,16 @@ Symfony2 Bundle for EventBand framework [![Build Status](https://travis-ci.org/e
 ## Adding event-band to a symfony2 project
 Add the following lines to your composer.json
 ```json
-"event-band/symfony-bundle": "dev-master",
-"event-band/amqplib-transport": "dev-master",
-"event-band/jms-serializer": "dev-master"
+"event-band/symfony-bundle": "~1.0",
```
Comment (korotovsky) [inline README.md]: Composer has "require" command. Maybe it's better to use this?

File: README.md [hunk 2]
```diff
@@ -107,4 +124,41 @@ sh$ app/cosole event-band:dispatch acme.echo.event
...
+    }
+
+    public function getMessage()
+    {
+        return $this->data['message'];
+    }
+}
+```
+### Config
+Add the following lines under 'event_band' section in your config.yml
```
Comment (korotovsky) [inline README.md]: `event_band`

File: README.md [hunk 3]
```diff
@@ -72,7 +89,7 @@ event_band:
 ### Adding band information to listener to make it asynchronous
 Add parameter "band" with name of consumer to event listener tag to show which consumer it belongs to.
```
Comment (korotovsky) [inline README.md]: `band`

Comment (coyl) [inline README.md]: oy vsyo

**Pull request merged**